packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.module.css

```diff
@@ -0,0 +1,37 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
```

Comment:
For `gap`, `.root` uses `2rem`, `.dialog` uses `var(--space--24)`, and `.footer` uses `var(--gap--16);`. Could we manage with one approach?